pandas/core/frame.py

```diff
@@ -1520,6 +1520,28 @@ def to_feather(self, fname):
         from pandas.io.feather_format import to_feather
         to_feather(self, fname)
 
+    def to_parquet(self, fname, engine, compression=None,
+                   **kwargs):
+        """
+        write out the binary parquet for DataFrames
+
+        .. versionadded:: 0.20.0
+
+        Parameters
+        ----------
+        fname : str
+            string file path
+        engine : parquet engine
```

Comment:
Either here, or as a followup, we could add a config option to control the default reader / writer. This will be like `io.excel.xls.writer`, so `io.parquet.writer/reader`?